transform/transformers/Transformer.py

```diff
@@ -0,0 +1,111 @@
+import logging
```

Comment:
has this only changed because the file name has changed?

Reply:
The existing code had a file called transformer and a class called Transformer . It seemed sensible to have class filenames match the classes ?